compiler/src/middle_end/linearize.re

```diff
@@ -1421,6 +1421,18 @@ let rec transl_anf_statement =
       | Exported => Global
       | Nonexported => Nonglobal
       };
+    let external_name =
+      switch (
+        List.find_opt(
```

Comment:
Why not use `fold_left`? This access pattern seems a bit unorthodox

Reply:
fold_left isn't _quite_ right here since it'll process the whole list without bailing, but I changed it anyway.